### PR TITLE
test(cli): add E2E tests for ui:create:react command

### PR DESCRIPTION
**Pull request by y-lakhdar**
https://coveord.atlassian.net/browse/CDX-173

### CODE REVIEW

File: packages/cli-e2e/utils/cli.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export async function setupUIProject(
   cliProcesses: ChildProcessWithoutNullStreams[],
   options: ISetupUIProjectOptionsArgs = {}
 ) {
-  const uiProjectFolderName = 'ui-projects';
+  const uiProjectFolderName = '../ui-projects';
```
Comment (y-lakhdar) [inline packages/cli-e2e/utils/cli.ts:64]: We need to move the ui projects outside the e2e-cli directory or we get warnings about different versions of Jest.

**Comment by louis-bompart** [comment]
Test looks good, I want to see this PR targeted on master tho before reviewing it for real 👀 ;)

**Comment by y-lakhdar** [comment]
Need to fix first CDX-190 first for the tests to pass

**Comment by y-lakhdar** [comment]
blocked by #111

**Review by olamothe — APPROVED** [review]

**Pull request merged**